test/e2e/driver/azuredisk_driver.go

```diff
@@ -63,16 +63,21 @@ func (d *azureDiskDriver) GetDynamicProvisionStorageClass(parameters map[string]
 	provisioner := d.driverName
 	generateName := fmt.Sprintf("%s-%s-dynamic-sc-", namespace, normalizeProvisioner(provisioner))
 	var allowedTopologies []v1.TopologySelectorTerm
-	if len(allowedTopologyValues) > 0 {
-		allowedTopologies = []v1.TopologySelectorTerm{
-			{
-				MatchLabelExpressions: []v1.TopologySelectorLabelRequirement{
-					{
-						Key:    TopologyKey,
-						Values: allowedTopologyValues,
+	if skuName, ok := parameters[consts.SkuNameField]; ok && strings.HasSuffix(skuName, "_ZRS") {
```

Comment:
Seems like we no longer need to handle ZRS in [func (pod *PodDetails) CreateStorageClass(...)](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/2082815881cd13577e43e701e95746b32a366241/test/e2e/testsuites/specs.go#L236) since it calls this function.